modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -1218,58 +1197,28 @@ public CompletableFuture<Void> alterTableAsync(String name, Consumer<TableChange
         }
     }
 
-    /** See {@link #alterTableAsync(String, Consumer)} for details. */
-    private CompletableFuture<Void> alterTableAsyncInternal(String name, Consumer<TableChange> tableChange) {
+    /** See {@link #alterTableAsync(String, Predicate)} for details. */
+    private CompletableFuture<Void> alterTableAsyncInternal(String name, Predicate<TableChange> tableChange) {
```

Comment:
Does it make sense to use `Function<TableChange, Boolean>` instead of `Predicate<TableChange>`? It may be unexpected that the predicate will change the passed argument